utils/src/test/java/com/cloud/utils/backoff/impl/ConstantTimeBackoffTest.java

```diff
@@ -43,7 +43,7 @@ public void run() {
         Thread.sleep(100);
         Assert.assertFalse(backoff.getWaiters().isEmpty());
         waitThread.interrupt();
-        Thread.sleep(100);
+        Thread.sleep(500);
```

Comment:
@davidjumani, how did you get to 500? I think it would be good to make it at least a symbolic value.
```suggestion
        final static int TIMEOUT_AFTER_INTERUPT = 500;
        Thread.sleep(TIMEOUT_AFTER_INTERUPT);
```